helm/Chart.yaml

```diff
@@ -19,3 +19,6 @@ dependencies:
     version: 8.12.3
     repository: https://kubernetes-charts.storage.googleapis.com
     condition: prometheus-operator.enabled
+  - name: cert-manager
+    version: 0.14.1
+    repository: https://charts.jetstack.io
```

Comment:
if this is going to require a bunch of setup stuff in LetsEncrypt and obviously adding network overhead, would it be a good idea to make this another toggle (`cert-manager.enabled`) and then we could even default it to `true` but generally make it easy to disable TLS encryption for quick testing and local dev?

Reply:
Agreed

Reply:
👍